snap/validate.go

```diff
@@ -47,6 +57,15 @@ func Validate(info *Info) error {
 		return err
 	}
 
+	epoch := info.Epoch
+	if epoch == "" {
+		return fmt.Errorf("snap epoch cannot be empty")
```

Comment:
Also added a call to `Validate()` from the store test to make sure these don't get out of sync.